arduino-ide-extension/src/browser/style/main.css

```
@@ -3,7 +3,8 @@
 }
 
 .p-TabBar-toolbar .item.arduino-tool-item {
-    margin-left: 3px;
+    margin-right: 8px;
+    margin-left: unset;
```

```suggestion
    margin: unset;
```

---

be careful here. @AlbyIanna unsetting the margin is different from unsettling the margin-left.

If the `margin-left` is assigned / inherited, `margin: unset;` will not clear it

---

You're right,  it's probably more correct to do like this:
```suggestion
    margin-left: 0;
```